src/widgets/WelcomeBlock/index.tsx

```diff
@@ -0,0 +1,2 @@
+import WelcomeBlock from "./WelcomeBlock";
+export default WelcomeBlock;
```

Comment:
if you get extra files in the folder? 
I suggest using aggregate export example: export * from './fileName'.